orchagent/vxlanorch.cpp

```diff
@@ -731,7 +750,24 @@ bool VxlanTunnelOrch::delOperation(const Request& request)
         return true;
     }
 
-    auto tunnel_term_id = vxlan_tunnel_table_[tunnel_name].get()->getTunnelTermId();
+    auto tunnel = vxlan_tunnel_table_[tunnel_name].get();
+
+    if (!tunnel->isMapperEmpty())
+    {
+        std::string vni_list;
+        bool first = true;
+        for (auto vni: tunnel->getMapperKeys())
+        {
+            if (!first)
+                vni_list += ", ";
+            vni_list += std::to_string(vni);
+            first = false;
+        }
+        SWSS_LOG_WARN("Vxlan tunnel '%s' has tunnel maps configured [%s]. Can't remove now.", tunnel_name.c_str(), vni_list.c_str());
+        return false;
```

Comment:
do we do retry here if mapper is not empty?

Reply:
if orchagent gets tunnel remove first and then tunnel map, then we should pend the tunnel remove (retry) later to resolve the dependency issue.

Reply:
'return false' mean don't erase from the queue. So the operation will be retried later

Reply:
i mean retry.